tests/TestFileWriterRotationScenarios/tst_FileWriterRotations.cpp

```diff
@@ -623,60 +634,66 @@ void FileWriterRotations::testRotateForTimePolicyAndSizeHigherNewer(int compress
-    QVERIFY(QFileInfo::exists(getCompressedFilename(filenames[1], compressionAlg)));
-    QVERIFY(QFileInfo::exists(filenames[2]));
-    QVERIFY(!QFileInfo::exists(getCompressedFilename(filenames[3], compressionAlg)));
+    QVERIFY(QFileInfo::exists(path+getCompressedFilename(filenames[0], compressionAlg)));
+    QVERIFY(QFileInfo::exists(path+getCompressedFilename(filenames[1], compressionAlg)));
+    QVERIFY(QFileInfo::exists(path+filenames[2]));
+    QVERIFY(!QFileInfo::exists(path+getCompressedFilename(filenames[3], compressionAlg)));
     if (compressionAlg != 0) { //verify that uncompressed files are not there
-            QVERIFY(!QFileInfo::exists(filenames[0]));
-            QVERIFY(!QFileInfo::exists(filenames[1]));
-            QVERIFY(!QFileInfo::exists(getCompressedFilename(filenames[2], compressionAlg)));
-            QVERIFY(!QFileInfo::exists(filenames[3]));
+            QVERIFY(!QFileInfo::exists(path+filenames[0]));
+            QVERIFY(!QFileInfo::exists(path+filenames[1]));
+            QVERIFY(!QFileInfo::exists(path+getCompressedFilename(filenames[2], compressionAlg)));
+            QVERIFY(!QFileInfo::exists(path+filenames[3]));
     }
-    QVERIFY(getCurrentLogFilename() == filenames[2]);
+    QVERIFY(getCurrentLogFilename() == getFileWithFullPath(filenames[2], path));
 
 
     setTestingCurrentDateTime(dtlist[1]);
     rotateFilesIfNeeded(); //or rotateFileForTimePolicy();
-    QVERIFY(!QFileInfo::exists(getCompressedFilename(filenames[0], compressionAlg)));
-    QVERIFY(QFileInfo::exists(getCompressedFilename(filenames[1], compressionAlg)));
-    QVERIFY(QFileInfo::exists(getCompressedFilename(filenames[2], compressionAlg)));
-    QVERIFY(QFileInfo::exists(filenames[3]));
+    QVERIFY(!QFileInfo::exists(path+getCompressedFilename(filenames[0], compressionAlg)));
+    QVERIFY(QFileInfo::exists(path+getCompressedFilename(filenames[1], compressionAlg)));
+    QVERIFY(QFileInfo::exists(path+getCompressedFilename(filenames[2], compressionAlg)));
+    QVERIFY(QFileInfo::exists(path+filenames[3]));
     if (compressionAlg != 0) { //verify that uncompressed files are not there
-            QVERIFY(!QFileInfo::exists(filenames[0]));
-            QVERIFY(!QFileInfo::exists(filenames[1]));
-            QVERIFY(!QFileInfo::exists(getCompressedFilename(filenames[3], compressionAlg)));
-            QVERIFY(!QFileInfo::exists(filenames[2]));
+            QVERIFY(!QFileInfo::exists(path+filenames[0]));
+            QVERIFY(!QFileInfo::exists(path+filenames[1]));
+            QVERIFY(!QFileInfo::exists(path+getCompressedFilename(filenames[3], compressionAlg)));
+            QVERIFY(!QFileInfo::exists(path+filenames[2]));
     }
-    QVERIFY(getCurrentLogFilename() == filenames[3]);
+    QVERIFY(getCurrentLogFilename() == getFileWithFullPath(filenames[3], path));
 
 
     foreach (QString s, filenames) {
-        deleteFile(getCompressedFilename(s, compressionAlg));
+        deleteFile(path + getCompressedFilename(s, compressionAlg));
```

Comment:
use `getFileWithFullPath` instead of `path + ...`

Reply:
done